extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -288,34 +314,82 @@ export class DataTierApplicationWizard {
 	}
 
 	public async deploy(): Promise<mssql.DacFxResult> {
+		const deployStartTime = new Date().getTime();
 		const service = await this.getService(msSqlProvider);
 		const ownerUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
 
-		return await service.deployDacpac(this.model.filePath, this.model.database, this.model.upgradeExisting, ownerUri, azdata.TaskExecutionMode.execute);
+		let result = await service.deployDacpac(this.model.filePath, this.model.database, this.model.upgradeExisting, ownerUri, azdata.TaskExecutionMode.execute);
+
+		// Deploy Dacpac: 'Deploy button' clicked in deploy summary page, Reporting the event selection to the telemetry
+		TelemetryReporter.createActionEvent(TelemetryViews.DataTierApplicationWizard, 'DataTierApplicationDeployDacpacOperation')
```

Comment:
this can probably be simplified to "DeployDacpacOperation" or "DeployDacpac" since the telemetry view is already specified. Same applies for all the other ones